gke/README.md

```diff
@@ -18,7 +18,9 @@ Instructions of the initial setup of the GCP project and GKE clusters for deploy
   cp config.yaml.tpl config.yaml
   ```
 
-* Add the GCP project id in `config.yaml`, **project** field.
+* Add the GCP project id used for deployment in `config.yaml`, **project** field.
+
+* Add the GCP project that hosts the Bigquery table in `config.yaml`, **store** field.
```

Comment:
Just curious... like the other configs in custom/ directory, should config.yaml be separated out as datacommons/ vs. custom/ ?

Reply:
This folder is for general setup and config.yaml is not checked in. The specific configs for each env is stored in https://github.com/datacommonsorg/mixer/tree/master/deploy/gke